frontend/src/components/WebhoncBadge/WebhoncBadge.tsx

```diff
@@ -0,0 +1,84 @@
+import { cn } from "@/utils";
+// import { Popover, PopoverContent, PopoverTrigger } from "../ui/popover";
+import {
+  HoverCard,
+  HoverCardContent,
+  HoverCardTrigger,
+} from "@radix-ui/react-hover-card";
+import { CheckIcon, CopyIcon } from "@radix-ui/react-icons";
+import { useState } from "react";
+import { Badge } from "../ui/badge";
+import { Button } from "../ui/button";
+import { useWebhoncConnectionId } from "./queries";
+
+export function WebhoncBadge() {
+  const [isCopied, setIsCopied] = useState(false);
+
+  const { data: url, isPending } = useWebhoncConnectionId();
+
+  const handleCopy = () => {
+    // copy to clipboard
+    navigator.clipboard.writeText(url!);
```

Comment:
nit: this doesn't handle the error case where text wasn't copied. we can refactor later on my branch since i have a utility hook for copying to clipboard